linkml/generators/docgen/common_metadata.md.jinja2

```diff
@@ -33,7 +33,9 @@ Instances of this class *should* have identifiers with one of the following pref
 | property | value |
 | --- | --- |
 {% for a in element.annotations -%}
-| {{a}} | {{ element.annotations[a].value }} |
+{% if a.first() == '_' -%}
```

Comment:
what's this?

Reply:
Fixed by using `a|string|first` since it's a jinja filter